docs/general/metrics.md

```diff
@@ -89,6 +90,27 @@ When introducing a new metric name check all existing schema files to make sure
 the name does not appear as a key of any "rename_metrics" section (keys denote
 old metric names in rename operations).
 
+### Metric attributes
+
+Metric attributes SHOULD follow the general [attribute naming rules](attribute-naming.md).
+In particular, metric attributes MUST have a namespace.
```

Comment:
We discussed previously that MUST is stronger that in the naming doc 

> All names that are part of OpenTelemetry semantic conventions SHOULD be part of a namespace.

I'd still prefer to keep it consistent

```suggestion
In particular, metric attributes SHOULD have a namespace.
```

Reply:
Ah I thought I have changed that. Sure, that makes sense.